lib/querystring.js

```diff
@@ -389,7 +387,7 @@ QueryString.parse = QueryString.decode = function(qs, sep, eq, options) {
     }
   }
 
-  Object.setPrototypeOf(obj, proto);
+  Object.setPrototypeOf(obj, Object.prototype);
```

Comment:
Without this the returned object has no prototype and won't have any of the Object.prototype methods (e.g. no toString). There's a non-zero chance of breaking someone.